.github/workflows/update-chain-specs.yml

```diff
@@ -0,0 +1,68 @@
+name: chain-specs-periodic-update
+
+on:
+  push:
+    branches: [ tka-update-chain-specs-ci ]   # TODO: remove before merging, I'm just testing
```

Comment:
When approved, I'll remove this line before merging. It's there just so that I can test if it work.